src/Memstate.Docs.GettingStarted/QuickStart/QuickStartTests.cs

```diff
@@ -10,40 +10,42 @@ namespace Memstate.Docs.GettingStarted.QuickStart
 {
     public class QuickStartTests
     {
-        private string Filename = "smoke_test_with_defaults";
-        private string JournalFile = "smoke_test_with_defaults.journal";
+        private string WireFileName = "smoke_test_with_defaults_wire";
+        private string WireJournalFile = "smoke_test_with_defaults_wire.journal";
+
+        private string JsonFileName = "smoke_test_with_defaults_json";
+        private string JsonJournalFile = "smoke_test_with_defaults_json.journal";
 
         [SetUp]
         [TearDown]
-        public void Setup()
+        public void SetupTeardown()
         {
-            Config.Reset().SerializerName = "Wire";
-            if (File.Exists(JournalFile)) File.Delete(JournalFile);
+            if (File.Exists(WireJournalFile)) File.Delete(WireJournalFile);
+            if (File.Exists(JsonJournalFile)) File.Delete(JsonJournalFile);
+            Config.Reset();
         }
 
-        // Hi all, here's some important notes on writing tests for .net core and standard,
-        // in case you want to create your own tests.
-        // https://github.com/nunit/docs/wiki/.NET-Core-and-.NET-Standard
-
         [Test]
-        public async Task Simple_end_to_end_getting_started_using_default_settings_with_filesytem_storage()
+         public async Task Simple_end_to_end_getting_started_using_default_wire_serializer_with_filesytem_storage()
         {
             Print("GIVEN I start a new Memstate engine for a LoyaltyDB using default settings");
             Print("   (using Wire format  & local filesystem storage)");
-            var settings = Config.Current.GetSettings<EngineSettings>();
-            settings.StreamName = Filename;
+            var config = Config.Current;
```

Comment:
I'm not familar with the new Config and settings syntax. Not sure if this is what you had in mind or what the simplest and least amount of code way of setting up is?